jbpm-bom/pom.xml

```diff
@@ -865,6 +865,16 @@
         <version>${version.org.jbpm}</version>
         <classifier>sources</classifier>
       </dependency>
+      <dependency>
+        <groupId>com.itextpdf</groupId>
+        <artifactId>itextpdf</artifactId>
+        <version>${version.org.itext.itextpdf}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.itextpdf.tool</groupId>
+        <artifactId>xmlworker</artifactId>
+        <version>${version.org.itext.itextpdf}</version>
+      </dependency>
```

Comment:
By convention, jbpm-bom should only contain dependencies with groupId `org.jbpm`. Would it possible to move this into kie-platform-bom (and ultimately to ip-bom)?